### PR TITLE
#3013 Outbound return SidePanel

### LINKED ISSUE #3013: Outbound returns detail view -- SidePanel
Part of https://github.com/msupply-foundation/open-msupply/issues/2920

Add SidePanel to page -- contents to be determined/discussed?

### PR DESCRIPTION
**Pull request by CarlosNZ**
Fixes #3013

# 👩🏻‍💻 What does this PR do? 
 

Thought I'd get this in quickly before I finish for the day.

Bare bones side panel here, basically copied the elements you'd included in your one @lache-melvin :)

# 🧪 How has/should this change been tested? 

Side panel opens and closes -- not much else :) 

## 💌 Any notes for the reviewer?

See inline

### CODE REVIEW

File: client/packages/invoices/src/Returns/DetailView/SidePanel/AdditionalInfoSection.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  //   const { colour, comment, user, update } = useOutbound.document.fields([
+  //     'colour',
+  //     'comment',
+  //     'user',
+  //   ]);
+  //   const [colorBuffer, setColorBuffer] = useBufferState(colour);
+  //   const [commentBuffer, setCommentBuffer] = useBufferState(comment ?? '');
+
+  //  TO-DO: Make this work
+  const { user } = data || {};
```
Comment (CarlosNZ) [inline client/packages/invoices/src/Returns/DetailView/SidePanel/AdditionalInfoSection.tsx:29]: I copied this from your one, but there's no `user` node on either of our ReturnByNumber queries (that I could tell). So not sure how you got the user data to be a valid type. Can probably figure out before merging, just wanted to rush this out the door for now.

**Review by lache-melvin — CHANGES_REQUESTED** [review]
Looks great Carl :) Just the user part of the query to update, up to you whether to add the delete now or later 🚀

**Comment by CarlosNZ** [comment]
@lache-melvin Righto, have updated the `user` node in the query. I see you've made an issue for delete lines mutation, so will leave that for now.

**Review by lache-melvin — APPROVED** [review]
Ah yep that issue is actually to delete lines within the return, not delete the return itself, but all good to get this in and do that separately!

**Pull request merged**